sample/applepay/index.html

```diff
@@ -69,12 +69,11 @@ <h1>PAY.JP Apple Pay on the Web サンプル</h1>
             amount: '1000'
           }
         };
-        var session = Payjp.applePay.buildSession(paymentRequest,
+        var session = payjp.applePay.buildSession(paymentRequest,
```

Comment:
```suggestion
        const session = payjp.applePay.buildSession(paymentRequest,
```

細かくて申し訳ない。 意図して `var` を採用している理由があれば無視で大丈夫です 🙋‍♂️

Reply:
見逃しでした。もらいます